daemon/mgr/cri_streaming.go

```diff
@@ -0,0 +1,50 @@
+package mgr
+
+import (
+	"fmt"
+	"io"
+	"net"
+
+	k8snet "k8s.io/apimachinery/pkg/util/net"
+	"k8s.io/client-go/tools/remotecommand"
```

Comment:
can we only import just k8s.io/client-go/tools/remotecommand? not the whole pkg?